backend/src/main/resources/application.yaml

```diff
@@ -1,3 +1,4 @@
+server.error.include-message: always
```

Comment:
I don't know about the ATO specifically but returning unfiltered error messages is generally frowned upon since you don't always know what's being returned by a given exception, and some random-seeming piece of info to you is someone else's attack vector (e.g. logging the Spring version number).

Agreed with @nullflux that we should keep this off by default and explicitly enable it where we do need it (specific methods, lower envs, etc.).

Reply:
Thanks! This is a good flag.

I'm not entirely sure how we'd go about explicitly enabling this only on the `UserAccountCreationController`, though - is there a way to turn on specific properties on a per-file basis? I'm thinking some kind of `@Value(server.error.include-message) = always` annotation, but I'm not sure if there's a way to do that in Spring.

Reply:
It doesn't appear that `server.error.include-message` can be scoped down in any way. I'm comfortable setting it to `always` provided that we implement a default `Exception.class` handler in `RestExceptionHandler` such that errors that _don't_ need a message don't get anything except an empty string

Reply:
Thanks, that's a good suggestion! I added a default `Exception` handler and verified that it still correctly throws known exceptions.